src/smockit/types.ts

```diff
@@ -24,14 +26,18 @@ export type MockReturnValue =
 export interface MockContractFunction {
   calls: string[]
 
+  reset: () => void
```

Comment:
Unused right now. Need to tweak how function mocking works. Will do this in the same PR as the one that updates how we handle call assertions.